packages/client-core/src/social/services/LocationService.ts

```diff
@@ -99,9 +98,6 @@ export const LocationState = defineState({
 
   socialLocationRetrieved: (location: LocationType) => {
     let bannedUsers = [] as string[]
-    location.locationBans.forEach((ban) => {
```

Comment:
we can also remove the other `bannedUsers` array creation, deduplication and removal from currentLocation state, as it is already unused